release-notes/formats.md

```diff
@@ -0,0 +1,101 @@
+# Release note formats
+
+Release notes are available in a combination of markdown and JSON formats. The intent is to make all release notes available in markdown and also in JSON if there is a  scenario that requires structured data. This content is [licensed](./license-information.md) for broad use.
+
+Our approach has evolved over time. The following descripions are for our latest practices.
+
+## Monthly patch release notes (markdown)
+
+We typically release an update each month. These often include security fixes for vulberabilities (AKA CVEs) disclosed on the same day.
+
+Examples:
+
+- [6.0.32](./6.0/6.0.32/6.0.32.md)
+- [8.0.1](./8.0/8.0.1/8.0.1.md)
+
+These markdown files include the following information:
+
+- Links to binaries
+- Notable changes, including CVEs
+- Compatibility information
+- Updates packages
+
+## Monthly preview release notes (markdown)
+
+We typically release a preview for the next major version each month. These are much the same as the monthly patches, however, will not include CVE information until the new major release is supported (starting with RC1).
+
+Examples:
+
+- [.NET 9 Preview 1](./9.0/preview/preview1/README.md)
+- [.NET 9 Preview 6](./9.0/preview/preview6/README.md)
+
+These markdown files are primarily composed of dense feature descriptions. This content is used as source material for [What's New](https://learn.microsoft.com/dotnet/core/whats-new/dotnet-9/overview) pages in official Microsoft docs, which are [often updated](https://github.com/dotnet/docs/pulls?q=is%3Apr+What%27s+New) on the same day as a preview release.
+
+Preview release notes are always in a `preview` folder. This approach was adopted so that preview releases do distract from stable releases once preview releases are no longer relevant.
```

Comment:
```suggestion
Preview release notes are always in a `preview` folder. This approach was adopted so that preview releases do not distract from stable releases once preview releases are no longer relevant.
```